tabular/src/autogluon/tabular/models/abstract/model_trial.py

```diff
@@ -60,15 +74,26 @@ def fit_and_save_model(model, params, fit_args, predict_proba_args, y_val, time_
     else:
         time_left = None
 
+    if epochs is None:
+        epochs = 1
+
     model.params.update(params)
     time_fit_start = time.time()
-    model.fit(**fit_args, time_limit=time_left, reporter=reporter)
-    time_fit_end = time.time()
-    y_pred_proba = model.predict_proba(**predict_proba_args)
-    time_pred_end = time.time()
-    model.val_score = model.score_with_y_pred_proba(y=y_val, y_pred_proba=y_pred_proba)
-    model.fit_time = time_fit_end - time_fit_start
-    model.predict_time = time_pred_end - time_fit_end
+    incremental = epochs > 1
+    for i in range(epochs):
+        model.fit(**fit_args, time_limit=time_left, reporter=reporter, warm_start=incremental)
```

Comment:
Incorrect time_limit specified. time_left is not valid anymore after the first loop.